pkg/crc/machine/libvirt/driver_linux.go

```diff
@@ -11,7 +12,12 @@ func CreateHost(machineConfig config.MachineConfig) *libvirt.Driver {
 
 	config.InitVMDriverFromMachineConfig(machineConfig, libvirtDriver.VMDriver)
 
-	libvirtDriver.Network = DefaultNetwork
+	if machineConfig.NetworkMode == network.VSockMode {
+		libvirtDriver.Network = ""
```

Comment:
When empty, do we remove the interface from the VM? If so, perhaps 'better' to annotate that here. 

Either we can choose to have that happen from the driver code by magic (side-effect behaviour) or explicitly do this.

Reply:
You can see it in a different way: when network is empty, there is no interface attached. For the libvirt case, we don't remove it, we just don't add the interface.

I will add a note here.

Reply:
oh sorry, 'Linux'. Still thinking of the first-serve Windows ;-).

But anyway, is empty not the default?

Reply:
yes it was just to make it obvious for the reader